### ISSUE TITLE
Settings | Team - Members block - Permissions

### ISSUE THREAD

**Issue by KostyaEver** [ever-co/ever-teams#916]
Managers can regulate members' permissions individually and for roles

For specifically selected Roles, manager case set permissions, OR create custom Role and ser custom permissions

<img width="1127" alt="Screenshot at May 24 15-29-22" src="https://github.com/ever-co/ever-gauzy-teams/assets/107203576/a18498de-a745-4b02-b401-0cb07898cb1d">

**Comment by evereq** [comment]
@Babajide777 I am not sure why it was designed all to be on one page... I think Members should be separate and there should be another page "Roles / Permissions" or something like that. Yes, it should be possible to select one of existed roles for each Member from Members page, but NOT to create new Role from that page. If user will click "Create" for Role, it should redirect user to page "Roles / Permissions" where s/he will create such Role and get back to Members page to assign that Role to some member, etc. I.e. I don't like idea to have it all integrated as usually those "Security" related things are better to keep separated etc.

**Comment by badalkhatri0924** [comment]
@evereq so should I work on this task or not? I've already made a separate page.

**Comment by evereq** [comment]
Yes, think we should do it, but for now we can allow to set Role for each member, no need for now ability to assign permissions to member! (I.e. permissions assigned to roles and role assigned to member)
